spec/integration/project_manager_spec.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require 'bolt_spec/integration'
+require 'bolt_spec/project'
+
+describe 'managing a project' do
+  include BoltSpec::Integration
+  include BoltSpec::Project
+
+  context 'creating a project' do
+    let(:command) { %w[project init myproject --modules puppetlabs-yaml] }
+
+    # Execute from a temporary project directory.
+    around(:each) do |example|
+      with_project do
+        Dir.chdir(project.path) do
```

Comment:
I'm wondering if we should just make this `Dir.chdir` part of `with_project` by default...

Reply:
Could be a separate `in_project` function that uses `with_project` then does the chdir